lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -102,9 +103,44 @@
           "type": "array",
           "items": {
             "type": "cid-link",
-            "description": "List of new blobs (by CID) referenced by records in this commit."
+            "description": "DEPRECATED -- will soon always be empty. List of new blobs (by CID) referenced by records in this commit."
           }
         },
+        "prevData": {
+          "type": "cid-link",
+          "description": "The root CID of the MST tree for the previous commit from this repo (indicated by the 'since' revision field in this message). Corresponds to the 'data' field in the repo commit object. NOTE: this field is effectively required for the 'inductive' version of firehose."
+        },
```

Comment:
I'm alright with this, but I do have an inclination to name it for its purpose rather than its content.  In particular, how the inductive checks work in the future might not be the same as they do today.  So I'd call it something like `check`, i.e. it's "the event's check CID".

Reply:
Note: we discussed today and decided to stick with `prevData`